test/Microsoft.Dnx.Testing.SampleTests/SampleTestFixture.cs

```diff
@@ -0,0 +1,28 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+
+namespace Microsoft.Dnx.Testing.SampleTests
+{
+    public class SampleTestFixture : IDisposable
+    {
+        public SampleTestFixture()
+        {
+            Console.WriteLine($@"
+Environment information:
+  DNX_HOME: {Environment.GetEnvironmentVariable("DNX_HOME")}
```

Comment:
This will be done when I deal with all the environment variables in another Issue